app-emulation/docker/docker-9999.ebuild

```diff
@@ -212,6 +215,9 @@ src_unpack() {
 		DOCKER_GITCOMMIT=$(git -C "${S}" rev-parse HEAD | head -c 7)
 		DOCKER_BUILD_DATE=$(git -C "${S}" log -1 --format="%ct")
 	fi
+	pushd "${S}"/components/engine || die
```

Comment:
Does `eapply` support `-d` to specify the directory?  Might be cleaner.

Reply:
It took a few tries, but yes it does, so long as you don't have a space... and it didn't work after the `--` either. Weird.